src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -60,6 +60,21 @@ function isTimeZoneIntialized() {
     return $('#timezone').val() !== TIMEZONE_SELECT_UNINITIALISED;
 }
 
+function addLeadingZeroes(num) {
+    return (num > 9 ? '' : '0') + num;
+}
+
+function displayUtcOffset(offset) {
+    if (offset === 0) {
+        return 'UTC';
+    }
+    const hr = Math.floor(Math.abs(offset / 60));
+    const min = Math.abs(offset) % 60;
+    // offset is calculated as the number of minutes needed to get to UTC
+    // thus the +/- sign needs to be swapped
+    return `UTC${offset < 0 ? '+' : '-'}${addLeadingZeroes(hr)}:${addLeadingZeroes(min)}`;
+}
+
```

Comment:
I think this would conflict with #8687 as well. Basically instructor feedback session has already been done there. Maybe you can copy it over to admin feedback.

Reply:
But over there, they are using the geography-based time zone, but you want to use offset-based time zone for admin session.